.github/workflows/popcorn_build_test.yml

```diff
@@ -10,15 +10,13 @@ on:
     branches: ["**"]
     paths:
       - "**"
-      - "!examples/**"
       - "!misc/**"
       - "!.github/**"
       - ".github/workflows/popcorn_build_test.yml"
   pull_request:
     branches: ["**"]
     paths:
       - "**"
-      - "!examples/**"
```

Comment:
These changes will trigger Popcorn build & test when an example changes. What's the point?